packages/functional-tests/pages/products/index.ts

```diff
@@ -23,6 +22,10 @@ export class SubscribePage extends BaseLayout {
     await frame.fill('.InputElement[name=cvc]', '444');
     await frame.fill('.InputElement[name=postal]', '77777');
     await this.page.check('input[type=checkbox]');
+  }
+
+  async clickPayNow() {
+    const frame = this.page.frame({ url: /elements-inner-card/ });
```

Comment:
Looks like we use this pattern 3&times; now, not sure if that qualifies as a potential candidate for refactoring.

```sh
git grep -in "this.page.frame" | cat

packages/functional-tests/pages/products/index.ts:9:     const frame = this.page.frame({ url: /elements-inner-card/ });
packages/functional-tests/pages/products/index.ts:19:    const frame = this.page.frame({ url: /elements-inner-card/ });
packages/functional-tests/pages/products/index.ts:28:    const frame = this.page.frame({ url: /elements-inner-card/ });
```